mautic/mautic
# Fix No Data shown for Most hit email redirects dashboard widget

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



This PR fixes no data shown for most hit email redirects dashboard widget.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Email 
3. Create template Email with link in it 
4. Go to campaign 
5. Create campaign with send email action 
6. Select above created email 
7. Publish the campaign 
8. Open mailbox read email and click on link on email 
9. Go to dashboard 
10. Add Widget -> 'Most hit Email redirects'
11. Save 
12. Widget should not stay blank




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11086"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
